### ISSUE TITLE
해당 프로필의 자동응답 시스템이 응답하지 않습니다.

### ISSUE THREAD

**Issue by gnsehfvlr** [plusfriend/auto_reply#506]
해당 프로필의 자동응답 시스템이 응답하지 않습니다.
라고 나오며 관리자 계정에는

 [관리자에게만 발송되는 메시지입니다] 사용자 봇서버에 에러가 감지 되었습니다. 
 내용:[잘못된 응답입니다. response json parsing error. user_key : drZvfF2REWZN, You'r server response : {
    "type" : "buttons",
    "buttons" : ["시작하기", "도움말"]
}]

이렇게 나옵니다. json 형식에 맞는데도 그러네요... 버그인가요??

웹 서버는 
http://45.120.65.8/kakaotalk_bot/keyboard
http://45.120.65.8/kakaotalk_bot/message 입니다.

참고로 제 3자가 카카오톡 봇 채팅창을 걸면 채팅창은 제대로 나오지만 답장을 하면 오류가 나옵니다..

# keyboard.php 파일입니다.
<?php
 
echo <<< EOD
{
  "type" : "buttons",
  "buttons" : ["시작하기", "도움말"]
}
EOD;

header('Content-type: application/json');

 
?>

# message.php 파일입니다.
<?php
 
// 요청을 받아 저장
$data = json_decode(file_get_contents('php://input'), true);
 
// 받은 요청에서 content 항목 설정
$content = $data["content"];
 
 
 
// '시작하기' 버튼 처리
if( $content == "시작하기" )
{
echo <<< EOD
    {
        "message": {
            "text": "아직 개발중이라 대답을 잘 못해도 이해해줘^^;"
        }
    }
EOD;
}
// '도움말' 버튼 처리
else if( $content == "도움말")
{
echo <<< EOD
    {
        "message": {
            "text": "이제 곧 정식 버전이 출시될거야. 조금만 기다려~~~"
        }
    }    
EOD;
}
// '안녕'이란 단어가 포함되었을때 처리
else if( strpos($content, "안녕") !== false )
{
echo <<< EOD
    {
        "message": {
            "text": "안녕~~ 반가워 ㅎㅎ"
        }
    }    
EOD;
}
// 그밖의 문장일때 
else
{
echo <<< EOD
    {
        "message": {
            "text": "나랑 놀자 ㅋㅋㅋ"
        }
    }    
EOD;
}
 
header('Content-type: application/json');

?>
 

뭐가 문제인가요 ㅠㅠ?

**Comment by gnsehfvlr** [comment]
네, 저도 HTTP 요청을 통해 응답 값이 잘 나오는데, 카카오톡 봇에서는 '해당 프로필의 자동응답 시스템이 응답하지 않습니다.'라고만 계속 나오며 동작이안되네요 ㅠ

**Comment by SerenityS** [comment]
@gnsehfvlr 아직 개발중이라~~ 뜨는데 아닌가요?

**Comment by SerenityS** [comment]
오류코드 봤는데 API 주소 입력에 keyboard까지 입력하셨는지 확인해보세요.

**Comment by SerenityS** [comment]
넵.ㅎㅎ
